generators/php/codegen/src/context/PhpAttributeMapper.ts

```diff
@@ -92,17 +123,20 @@ export class PhpAttributeMapper {
                 });
             }
             case "union": {
-                return php.instantiateClass({
-                    classReference: this.context.getUnionClassReference(),
-                    arguments_: type.internalType.types.map((unionType) =>
-                        this.getArrayTypeAttributeArgument(unionType)
-                    )
-                });
+                const unionTypeParameters = this.getUnionTypeParameters(type.internalType.types);
+                // dedupe in getUnionTypeParameters could result in a single value
```

Comment:
nit: I'd remove this comment since it's already fairly clear from the code. This might apply elsewhere.

Reply:
done!